### ISSUE TITLE
Bug when opening a folder

### ISSUE THREAD

**Issue by petacreepers23** [PowerShell/PSReadLine#1470]
Don't know very well what caused this, only happened in one folder, but lots of times. Here is the report:

```
Últimas 1 claves:
 .

Excepción:
System.ArgumentOutOfRangeException: El valor debe ser mayor que o igual a cero y menor que el tamaño de búfer de consola en dicha dimensión.
Nombre del parámetro: left
Valor actual -1.
   en System.Console.SetCursorPosition(Int32 left, Int32 top)
   en Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   en Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   en Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   en Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   en Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   en Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   en Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   en Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
```

**Comment by iSazonov** [comment]
See #1306

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.1 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions.
